### PR TITLE
🔥 Added cool README.md

### PR DESCRIPTION
**Pull request by PiyushKhurana**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,48 @@
...
+
+<!-- CONTRIBUTING -->
+## Contributing
+
+Contributions are what make the open source community such an amazing place to be learn, inspire, and create. Any contributions you make are **greatly appreciated**.
+
+To contribute to project, follow these steps:
+
+1. Fork this repository.
+2. Create a branch: `git checkout -b <your_branch_name>`.
```
Comment (NikhilJoshua) [inline README.md:43]: Branch name ideally should be your-name/(feature or fix) What do you think @PiyushKhurana ?

**Review by NikhilJoshua — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (PiyushKhurana) [reply README.md:43]: Yep I agree 👍

Comment (NikhilJoshua) [reply README.md:43]: Never mind, forking will show the name. Sorry my bad. :)

**Review by NikhilJoshua — APPROVED** [review]

**Comment by NikhilJoshua** [comment]
Great work @PiyushKhurana absolutely loved it.

**Comment by PiyushKhurana** [comment]
✌️ I love working on these kind of stuffs

**Pull request merged**